crates/primitives/src/transaction/pooled.rs

```diff
@@ -10,6 +10,9 @@ use derive_more::{AsRef, Deref};
 use reth_codecs::add_arbitrary_tests;
 use serde::{Deserialize, Serialize};
 
+#[cfg(feature = "optimism")]
+use crate::TxDeposit;
```

Comment:
This looks to only be used once, we should remove this import and reference via absolute path inline.

Reply:
Addressed in https://github.com/anton-rs/op-reth/pull/154